src/app/public/shared/info/info.component.ts

```diff
@@ -1,15 +1,13 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-info',
   templateUrl: './info.component.html',
-  styleUrls: ['./info.component.css']
+  styleUrls: ['./info.component.css'],
 })
-export class InfoComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
+export class InfoComponent {
+  repos: number = 110;
+  gists: number = 40;
+  followers: number = 3240;
+  following: number = 210;
```

Comment:
in fact... other question: you investigate the API response for see which are the properties name that it will return??? Maybe it's a good idea to tackle this in a meeting 👍

Reply:
changed! :)